src/pages/apps/[id]/environments/[env-id]/deployments/runtime-logs/RuntimeLogs.tsx

```diff
@@ -8,21 +8,10 @@ import { useFetchDeploymentRuntimeLogs } from "./actions";
 import Spinner from "~/components/Spinner";
 import InfoBox from "~/components/InfoBoxV2";
 import Link from "~/components/Link";
+import Button from "~/components/ButtonV2";
 
 const renderLog = (log: Log, i: number) => {
   let data = log.data.split(/END\sRequestId:/)[0];
-
```

Comment:
we could convert this into a feature, but we shouldn't filter logs like this on FE for every project.